ocelot/adaptors/genesis.py

```diff
@@ -3056,6 +3056,7 @@ def transform_beam_twiss(beam, transform=None, s=None):
         else:
             idx = find_nearest_idx(beam.s, s)
 
+        # remark C. Lechner, 2024-04: when running pytest, the use of numpy.matrix in this function raises "PendingDeprecationWarning: the matrix subclass is not the recommended way to represent matrices or deal with linear algebra (see https://docs.scipy.org/doc/numpy/user/numpy-for-matlab-users.html). Please adjust your code to use regular ndarray."
```

Comment:
ok then why not replace with np.array?

Reply:
I agree that replacing these with `np.array` and the related matrix operations is what needs to be done.
But one would need to study the code a bit since the objects generated in `transform_beam_twiss` are passed to function `find_transform`. In addition, before changing this tests for the current behavior need to be implemented to avoid bugs.